vermouth/processors/name_moltype.py

```diff
@@ -0,0 +1,85 @@
+# Copyright 2018 University of Groningen
```

Comment:
Missing python shebang

Reply:
Shebangs are only relevant for things you will call from your shell.